pkg/management/component_test.go

```diff
@@ -1596,6 +1596,18 @@ func TestExecuteActionsEmail(t *testing.T) {
 
 		assert.NotNil(t, err)
 	}
+	// Send email actions, but not sms-password-set
+	{
+
```

Comment:
`var actions2` and `var reqActions2` are only used here... they could be declared here

Reply:
Thank you for the suggestion
DONE